lmdk/cmake/ldscripts/bss_linker_script.txt

```diff
@@ -16,5 +16,18 @@ SECTIONS {
         _next_end = ABSOLUTE(.);
         *(.bss)
         *(.bss.*)
+
+        /* Heap memory */
+        . = ALIGN(4);
+        __stack = .;
```

Comment:
why stack?

Reply:
Xtensa implementation of the `_sbrk_r` use `__stack` symbol to check that a new allocated memory from a heap not overlaps with stack. Without this symbol it simply won't link successfully.